client/python/lib/ovmsclient/tfs_compat/http/serving_client.py

```diff
@@ -103,21 +103,27 @@ def get_model_metadata(self, request):
             >>> response = client.get_model_metadata(request)
             >>> type(response)
         '''
-
-        HttpClient._check_model_metadata_request(request)
-
+        self._validate_timeout(timeout)
+        request = make_metadata_request(model_name, model_version)
```

Comment:
I think we need to change arguments provided to `get_model_status` here. Currently we provide request, but we should provide same arguments as in grpc implementation.

Reply:
What is wrong with this one? I don't see any issue with this part.

Reply:
@mzegla highlighted part was indeed correct, however I could not highlight `get_model_status` function declaration, which was not changed and that was the problem. Everything is fine now.